Applications/Calculator/CalculatorWidget.cpp

```diff
@@ -197,3 +197,46 @@ void CalculatorWidget::update_display()
     else
         m_label->set_text("");
 }
+
+void CalculatorWidget::keydown_event(GKeyEvent& event)
+{
+    RefPtr<GButton> clicked_button;
```

Comment:
Well it's certainly not clicked. Maybe it's `button_to_activate` or something like that.

Reply:
(And no need to keep it in a `RefPtr` here, a raw pointer will do.)